src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/Unicode/GeneratorHelper.cs

```diff
@@ -10,24 +10,14 @@ namespace System.Text.RegularExpressions.Symbolic.Unicode
     [ExcludeFromCodeCoverage]
     internal static class GeneratorHelper
     {
-        public static void WriteInt64ArrayInitSyntax(StreamWriter sw, long[] values)
-        {
-            sw.Write("new long[] {");
-            for (int i = 0; i < values.Length; i++)
-            {
-                sw.Write($" 0x{values[i]:X}, ");
-            }
-            sw.Write("}");
-        }
-
         public static void WriteByteArrayInitSyntax(StreamWriter sw, byte[] values)
```

Comment:
perhaps this could now get inlined into WriteIgnoreCaseBDD.

Reply:
Jose's change already moved it to a different file. But it's still called in multiple places.